src/components/comments/CommentForm.jsx

```diff
@@ -11,16 +11,35 @@ import { LOCAL_HOST } from "../../consts";
 class CommentForm extends Component {
   constructor(props) {
     super(props);
-    this.state = { userID: null, userName: '', isNewUser: false };
-    // check if user record exist in localStorage
-    if (localStorage.getItem('user') != null) {
-      let storedUser = JSON.parse(localStorage.getItem('user'))
-      this.state.userID = storedUser.userID
-      this.state.userName = storedUser.userName
-      this.setState(storedUser)
+    this.state = { userID: null, userName: '' };
+    /**
```

Comment:
Don't use JavaDoc comments in places like this, it's bad styling. Just stick to //